doc/UsersGuide/source/ContributingDevelopment.rst

```diff
@@ -4,13 +4,201 @@
+| 	      |                                                              |  4.6.3  |                         |
++             +                                                              +         +                         +
+| 	      |                                                              |  4.7.0  |                         |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+| 	      |                                                              |  1.8.14 |                         |
++             +                                                              +         +                         +
+|HDF5         | https://www.hdfgroup.org/downloads/hdf5/                     |  1.8.16 | $HDF5                   |
++             +                                                              +         +                         +
+| 	      |                                                              |  1.10.1 |                         |
++             +                                                              +         +                         +
+| 	      |                                                              |  1.10.4 |                         |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+|jasper       | https://github.com/mdadams/jasper                            | 1.900.1 | $JASPER_LIB             |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+|zlib 	      | https://github.com/madler/zlib                               | 1.2.11  | $Z_LIB                  |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+|libpng       | http://www.libpng.org/pub/png/libpng.html                    | 1.2.44  | $PNG_LIB                |
++-------------+--------------------------------------------------------------+---------+-------------------------+
+
+
+Compiler:
+
+
++--------------------+------------------------------------------------------------------+
+|Compiler name       |    Branch name                                                   |
++====================+==================================================================+
+| Intel              |   16.0.3.210,16.3.210,18.0.1.163, 18.0.3.222,18.0.5.274, 19.0.2  |
++--------------------+------------------------------------------------------------------+
+|impi                |   2018.0.4, 2018.4.274,18.0.1, 2019.2.187                        |
++--------------------+------------------------------------------------------------------+
+
+
+These libraries are available on several platforms including Hera, Cheyenne and WCOSS. Installing these libraries on jet and gaea is a work in progress. Users who need to build the libraries. [https://github.com/NOAA-EMC/NCEPLIBS/wiki/Cloning-and-Compiling-NCEPLIBS]
```

Comment:
Please revise this so it is a complete sentence

Reply:
Done